frontend/cypress/tests/codeView/codeView_hideFileDetails.js

```diff
@@ -66,6 +66,42 @@ describe('hide all file details', () => {
         .should('not.be.visible');
   });
 
+  it('check file can be hidden after scrolling', () => {
+    cy.get('.icon-button.fa-code')
+        .should('be.visible')
+        .first()
+        .click();
+
+    cy.get('#tab-authorship .files', { timeout: 90000 })
+        .should('be.visible');
+
+    // contents of the first file should be visible
+    cy.get('#tab-authorship .file-content ')
+        .first()
+        .should('be.visible');
+
+    // scroll some lines in the contents of the first file
+    cy.get('#tab-authorship .file-content ')
+        .first()
+        .scrollIntoView({ offset: { top: 500, left: 0 } });
+
+    // close contents of the first file
+    cy.get('#tab-authorship .title .caret')
+        .should('be.visible')
+        .first()
+        .click();
+
+    // contents of the first file should not be visible
+    cy.get('#tab-authorship .file-content ')
+        .first()
+        .should('not.be.visible');
+
+    // title of the first file should still be visible
+    cy.get('#tab-authorship .file ')
+        .first()
+        .should('be.visible');
+  });
```

Comment:
@ckcherry23 Thanks for the work! Just a small suggestion. Is it possible to add the case when you scroll a bit without collapsing the file and check that the title is still visible. Another case will be scrolling back to the previous file (without reaching the top of the file) should display the title of the file.

Reply:
Hi @zhoukerrr, I've merged your suggested test cases in the same Cypress test. Hope this is okay?